tests/diff/patchid.c

```diff
@@ -22,7 +22,7 @@ void test_diff_patchid__simple_commit(void)
 
 void test_diff_patchid__filename_with_spaces(void)
 {
-	verify_patch_id(PATCH_APPEND_NO_NL, "f0ba05413beaef743b630e796153839462ee477a");
+	verify_patch_id(PATCH_APPEND_NO_NL, "cc0d67fc516f3378baa8a72b248da6e26c5dbf4a");
```

Comment:
This is incorrect, though. git-patch-id(1) prints "f0ba05413beaef743b630e796153839462ee477a", so we cannot simply alter the expected ID here. So this indicates that we have to adjust how we calculate the patch ID when there's no trailing newline. Just let me know whether you want to amend that or whether I should take over again :)

Reply:
yes, please take over. probably better someone fixes this that knows what the issue is ^^